### ISSUE TITLE
Uncaught TypeError: Cannot read property 'on' of null

### ISSUE THREAD

**Issue by abm0** [samu/git-diff-details#27]
[Enter steps to reproduce below:]

1. Open folder as project, or remove project folder

**Atom Version**: 1.0.3
**System**: Mac OS X 10.10.3
**Thrown From**: [git-diff-details](https://github.com/samu/git-diff-details) package, v0.19.0

### Stack Trace

Uncaught TypeError: Cannot read property 'on' of null

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/git-repository.js:133

TypeError: Cannot read property 'on' of null
    at GitRepository.module.exports.GitRepository.onDidChangeStatuses (/Applications/Atom.app/Contents/Resources/app.asar/src/git-repository.js:133:26)
    at AtomGitDiffDetailsView.module.exports.Housekeeping.subscribeToRepository (/Users/i-dranichnikov/.atom/packages/git-diff-details/lib/housekeeping.coffee:107:41)
    at /Users/i-dranichnikov/.atom/packages/git-diff-details/lib/housekeeping.coffee:42:26
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
    at Project.module.exports.Project.addPath (/Applications/Atom.app/Contents/Resources/app.asar/src/project.js:274:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/atom.js:908:41
    at EventEmitter.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/atom.js:444:16)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:166:7)
```

### Commands

```
     -4:04.1.0 editor:select-to-beginning-of-word (atom-text-editor.editor.is-focused)
     -3:59.7.0 linter:set-bubble-transparent (atom-text-editor.editor.is-focused)
     -3:59.4.0 pane:show-next-item (atom-text-editor.editor.is-focused)
     -3:55.5.0 linter:set-bubble-transparent (atom-text-editor.editor.is-focused)
     -3:55.4.0 pane:show-previous-item (atom-text-editor.editor.is-focused)
     -3:54.8.0 linter:set-bubble-transparent (atom-text-editor.editor.is-focused)
     -3:54.2.0 pane:show-next-item (atom-text-editor.editor.is-focused)
     -3:50.7.0 pane:show-previous-item (atom-text-editor.editor.is-focused)
     -3:49.6.0 core:move-left (atom-text-editor.editor.is-focused)
     -3:49.4.0 linter:set-bubble-transparent (atom-text-editor.editor.is-focused)
  3x -3:49.3.0 editor:select-to-end-of-word (atom-text-editor.editor.is-focused)
  2x -3:46.9.0 core:backspace (atom-text-editor.editor.is-focused.autocomplete-active)
  3x -3:45.2.0 core:save (atom-text-editor.editor.is-focused)
     -3:22.4.0 pane:show-previous-item (atom-text-editor.editor.is-focused)
     -3:19.8.0 core:copy (atom-text-editor.editor.is-focused)
  2x -0:12.5.0 application:add-project-folder (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
```

### Config

```json
{
  "core": {
    "themes": [
      "isotope-ui",
      "atom-dark-syntax"
    ],
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ],
    "projectHome": "~/rails_projects",
    "disabledPackages": [
      "open-on-github",
      "autocomplete-css",
      "goto",
      "jslint",
      "csslint"
    ]
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.3.0
autocomplete-ruby, v0.1.0
color-picker, v2.0.11
docs-snippets, v0.8.0
emmet, v2.3.12
git-blame, v0.4.5
git-diff-details, v0.19.0
isotope-ui, v2.5.1
jumpy, v2.0.9
language-haml, v0.21.0
lesscompile, v0.5.0
linter, v1.2.4
linter-ruby, v0.1.6
pain-split, v1.4.0
project-manager, v1.15.11

# Dev
No dev packages
```

**Comment by bisraelsen** [comment]
+1
**Atom Version:** 1.0.4
**System:** Xubuntu 14.04
**Thrown From:** git-diff-details package, v0.19.0

**Comment by samu** [comment]
@abm0 @bisraelsen this should be fixed with `v0.20.0`
Please let me know if it works for you!

**Comment by abm0** [comment]
I'm not getting this error anymore, thanks!
